README.md

```diff
@@ -1,2 +1,209 @@
+4. Run
+   ```
+   npx hint .
+   ```
+5. Fix validation errors.
+
+### [Stylelint](https://stylelint.io/)
+
+A mighty, modern linter that helps you avoid errors and enforce conventions in your styles.
+
+1. Run
+
+npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+not sure how to use npm? Read this.
+
+2. Copy .stylelintrc.json to the root directory of your project.
+
+3. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
+
+If you think that change is necessary - open a Pull Request in this repository and let your code reviewer know about it.
+4. Run npx stylelint "**/*.{css,scss}" on the root of your directory of your project.
+
+5. Fix linter errors.
+
+6. **IMPORTANT NOTE:** feel free to research auto-correct options for Stylelint if you get a flood of errors but keep in mind that correcting style errors manually will help you to make a habit of writing a clean code!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: (https://github.com/joyapisi)
+- Twitter: (https://twitter.com/joyapisi)
+- LinkedIn: (https://http://www.linkedin.com/in/joy-phoebe-00b80a13a)
+
+
+👤 **Author2**
+ 
+- GitHub:(https://github.com/cindykandie)
+
+
+
+👤 **Author3**
+ 
+- GitHub:(https://github.com/joyapisi)
+
+## 🤝 Contributing <a name="contributing"></a>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[new_feature_1]**
+This project will not have any additional features.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, kindly leave a comment below and share it with someone who enjoys coding! Coding is all about continuous learning and allowing yourself to be a beginner. Keep going! 
+
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="Microverse Inc."></a>
+
+I'm thankful to Microverse for providing a study platform which guided me through this project and to my morning-session team at Microverse for the team effort in doing this together. 
+
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+- **[Question_1]**
+Do I have to use the gitbash terminal?
+
+  - [Answer_1]
+You can use other terminals for this project as long as your commands are correct.
+
+- **[Question_2]**
+Where can I download node JS for installation?
+
+  - [Answer_2]
+Node Js can be downloaded here- https://nodejs.org/en/download/  
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] I would advise you to resolve the link to MIT because it is upon it your repo derives a legal existence. At the moment, it leads to a 404 page. Kindly fix it.